erpnext/controllers/queries.py

```diff
@@ -10,7 +10,7 @@
 from erpnext.stock.get_item_details import _get_item_tax_template
 from frappe.utils import unique
 
- # searches for active employees
+# searches for active employees
```

Comment:
Create this PR on `bs-jun-sprint-2` branch

Reply:
@shruti2323 the `bs-jun-sprint-2` branch is already released now, if the code is not present in `bs-jul-sprint-1` branch, then createt a new PR on `pre-release` branch.

Reply:
This PR is already rebased.